core/common/src/MapLayerSettings.ts

```diff
@@ -349,24 +408,102 @@ export class MapLayerSettings {
 
     return children;
   }
+
+  /** @internal */
+  protected static mapTypeName(type: BackgroundMapType) {   // TBD.. Localization.
```

Comment:
Shouldn't let user pass an optional name to `BaseMapLayerSettings.fromProvider`, otherwise default to english?  Currently `BaseMapLayerSettings.fromProvider` is mainly use to set the base map, and I'm wondering if there is much use case to display the base map name to the end-user?

Reply:
@RBBentley @pmconne

Reply:
The problem is this is done completely behind the user's back - they have no opportunity to provide localization. See BackgroundMapProvider.fromBackgroundMapProps, which is invoked by BackgroundMapSettings' constructor.
Definitely a problem for a separate PR.